src/components/Autocomplete.jsx

```diff
@@ -20,17 +20,31 @@ const Autocomplete = ({ id, suggest, label, ...rest }) => {
           {label}
         </label>
       )}
-      <UkAutocomplete
-        id={id}
-        className="dg_form-field_input--text"
-        source={handleSource}
-        showNoOptionsFound={false}
-        templates={{
-          inputValue: FormatAddress,
-          suggestion: FormatAddress,
-        }}
-        {...rest}
-      />
+      <div style={{ position: "relative" }}>
```

Comment:
This component is a good candidate for it's own library or to be part of components. Need to document this somewhere, we should come back to this before it's approved.